tools/azure-sdk-tools/packaging_tools/templates/README.md

```diff
@@ -1,5 +1,4 @@
-Microsoft Azure SDK for Python
-==============================
+# Microsoft Azure SDK for Python
```

Comment:
Let's take this opportunity to remove the two "if is_arm" and their content. ARM is de-facto standard nowadays, this content doesn't matter anymore